label_studio/io_storages/base_models.py

```diff
@@ -221,17 +410,28 @@ def sync(self):
         if redis_connected():
             queue = django_rq.get_queue('low')
             meta = {'project': self.project.id, 'storage': self.id}
-            if not is_job_in_queue(queue, "sync_background", meta=meta) and \
-                    not is_job_on_worker(job_id=self.last_sync_job, queue_name='default'):
-                job = queue.enqueue(sync_background, self.__class__, self.id,
-                                    meta=meta)
-                self.last_sync_job = job.id
-                self.save()
-                # job_id = sync_background.delay()  # TODO: @niklub: check this fix
-                logger.info(f'Storage sync background job {job.id} for storage {self} has been started')
+            if (
+                    not is_job_in_queue(queue, "import_sync_background", meta=meta) and
+                    not is_job_on_worker(job_id=self.last_sync_job, queue_name='low')
+            ):
+                sync_job = queue.enqueue(
```

Comment:
Can we do it with our general `start_job_async_or_sync` function? I like the idea of having this `if redis_connected` check only in one place in the project.

Reply:
it was done not by me, but I think it's because of 
```
 if (
                    not is_job_in_queue(queue, "import_sync_background", meta=meta) and
                    not is_job_on_worker(job_id=self.last_sync_job, queue_name='low')
            ):
```
in start_job_async_or_sync we don't have this check..